src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final double DEFAULT_RADIUS = 10.0;
+    private static final double MAX_RANDOM_VALUE = 10.;
+    private static final int AMOUNT_OF_FIGURES = 4;
```

Comment:
The class name `FigureSupplier` is good as it gives a clear indication of its responsibility. However, the constant `AMOUNT_OF_FIGURES` should be `FIGURE_COUNT` to follow the checklist's example for constant naming, which enhances code readability.